### PR TITLE
refactor[yaml]: Updated the cassandra application specific pipeline

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta aman.gupta@mayadata.io

1. Removed some of the unwanted chaos jobs in cassandra branch.
2. Updated some chaos jobs based on stroage engines i.e. cstor and jiva.
3. Resolve typos and linting errors.
4. Modified the create_striped_pool file in setup stage.
5. Added cleanup for cassandra-jiva.
5. Updated the .gitlab-ci.yaml based on above mentioned modifications.

### CODE REVIEW

File: Openshift-EE/pipelines/Cassandra/stages/2-setup/OJEF-create-striped-pool/create-pool [hunk 1]
```diff
@@ -55,17 +72,18 @@ bash Openshift-EE/utils/event_updater jobname:cstor-pool $test_name jobid:"$job_
...
 
-
 ## Check the test status & result from the litmus result custom resource
 source ~/.profile
+
 gittoken=$(echo "$github_token")
+
 testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-columns=:spec.testStatus.result)
-#python3 Openshift-EE/utils/result/result_update.py $job_id OJEF 2-setup "Create cStor based Striped storage pool on disks." $testResult $pipeline_id "$current_time" $commit_id $gittoken
+#python3 Openshift-EE/utils/result/result_update.py $job_id HC5P 2-setup "Create cStor based Striped storage pool on disks and corresponding storage class" $testResult $pipeline_id "$current_time" $commit_id $gittoken
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/Cassandra/stages/2-setup/OJEF-create-striped-pool/create-pool]: @w3aman please remove this commented lines.

File: Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay [hunk 2]
```diff
@@ -30,44 +30,45 @@ current_time=$(eval $time)
...
 
+echo "*******Performing network delay on Target**********"
+## testcase name is defined in test_vars of LitmusBook.
 run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-target-network-delay metadata=${run_id})
 echo $test_name
 
 cd litmus
-cp experiments/chaos/openebs_target_network_delay/run_litmus_test.yml run_controller_delay_test_jiva.yml
+# copy the content of application pod failure chaos `run_litmus_test.yml` into a different file to update the test specific parameters.
+cp experiments/chaos/openebs_target_network_delay/run_litmus_test.yml run_targetdelay_test.yml
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay]: @w3aman use the file name like 'run_target_delay_test.yml`

File: Openshift-EE/pipelines/Cassandra/stages/5-cleanup/5N53-cassandra-jiva-cleanup/5N53-delete-cassandra-jiva [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+present_dir=$(pwd)
+echo $present_dir
+
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/pooling jobname:delete-cassandra-cstor
+bash Openshift-EE/utils/e2e-cr jobname:delete-cassandra-jiva jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+#### Deleting Cassandra application
+run_id="jiva_delete"; test_name=$(bash Openshift-EE/utils/generate_test_name testcase=cassandra-deployment metadata=${run_id})
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/Cassandra/stages/5-cleanup/5N53-cassandra-jiva-cleanup/5N53-delete-cassandra-jiva]: use run_id `jiva-delete`

File: Openshift-EE/pipelines/Cassandra/stages/2-setup/OJEF-create-striped-pool/create-pool (hunk 1 above)
Comment (w3aman) [reply Openshift-EE/pipelines/Cassandra/stages/2-setup/OJEF-create-striped-pool/create-pool]: Done!

File: Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay (hunk 2 above)
Comment (w3aman) [reply Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay]: Okay!

File: Openshift-EE/pipelines/Cassandra/stages/5-cleanup/5N53-cassandra-jiva-cleanup/5N53-delete-cassandra-jiva (hunk 3 above)
Comment (w3aman) [reply Openshift-EE/pipelines/Cassandra/stages/5-cleanup/5N53-cassandra-jiva-cleanup/5N53-delete-cassandra-jiva]: Done!

File: Openshift-EE/pipelines/Cassandra/stages/5-cleanup/5N53-cassandra-jiva-cleanup/5N53-delete-cassandra-jiva [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+ | appLabel          | env    | app=cassandra                       | app=cassandra-jiva                                   |
+ | pvcName           | env    | openebs-cassandra                   | openebs-cassandra-jiva                               | 
+ | storage class     | env    | openebs-standalone                  | openebs-jiva-default                            |
+ | appNamespace      | env    | app-cass-ns                         | app-cass-ns-jiva                                     |
+ | action            | env    | provision        		                | deprovision                                     |
+ ----------------------------------------------------------------------------------------------------------------------
+EOF
+# Replacing the required parameters in the litmus experiment.
+
+sed -i -e 's/value: openebs-standalone/value: openebs-cstor-sparse/g'\
```
[the remark this answers is not in the document]
Comment (w3aman) [reply Openshift-EE/pipelines/Cassandra/stages/5-cleanup/5N53-cassandra-jiva-cleanup/5N53-delete-cassandra-jiva]: changed !

**Pull request merged**